src/main/java/nextstep/subway/section/domain/Distance.java

```diff
@@ -0,0 +1,52 @@
+package nextstep.subway.section.domain;
+
+import java.util.Objects;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+
+import nextstep.subway.exception.SubwayLogicException;
+
+@Embeddable
+public class Distance {
+
+	private static final String DISTANCE_INVALID_MESSAGE = "distance는 0이상입력 해주세요";
+	@Column(name = "distance")
```

Comment:
중요한 부분은 아니지만, 문맥이 달라지는 부분에서 줄바꿈을 해주시면 가독성이 올라갈 것 같아요.

```suggestion
	private static final String DISTANCE_INVALID_MESSAGE = "distance는 0이상 입력 해주세요";
	
	@Column(name = "distance")
```